### ISSUE TITLE
Discuss:: Set Minimum contribution based on expected Repayment

### ISSUE THREAD

**Issue by adibas03** [enabledao/enable-contracts#53]
While setting up the e2e tests, I realized, that based on the size of the contribution to the loan, some contributors might not have a repayment due every month, but instead, every second month or more.
For instance: A contribution of *6* units out of *60,000* units (0.01%), will not be due for any repayment until *10,000* units have been repaid.
Do we leave this as is and educate, or do we set a minimum contribution, based on the expected repayment.
I think education might be sufficient to allow for smaller contribution, especially since the project is P2P and less corporate.
Note* *Implementation of any decision should be against the next version*

@onggunhao @tspoff thoughts

**Comment by onggunhao** [comment]
@adibas03 I'm strongly in favor of a minimum contribution of 20 Dai. 

20 Dai is also the minimum sum for PoolTogether. Their reason was gas fees

**Comment by adibas03** [comment]
I think 100 DAI might be a good place, although 50 DAI might also work, though 100 DAI ensures we have fewer contributors

**Comment by onggunhao** [comment]
Crazy idea: what if we set it to 1,000 Dai? We would have to find 60 people willing to do that

**Comment by adibas03** [comment]
That would be great, but according to the data from our research, 100 DAI is the sweet spot

**Comment by onggunhao** [comment]
Yeah that's true. Talking about that, we should do a follow-up survey on Twitter to measure whether that sentiment has changed
